policyengine_us/parameters/gov/states/ri/tax/income/exemption/amount.yaml

```diff
@@ -2,6 +2,7 @@ description: Rhode Island provides the following amount for each exemption in th
 values:
   2021-01-01: 4_250
```

Comment:
could you check all the file that adjust every year if we need updates and rounding?